wagtail/test/testapp/models.py

```diff
@@ -977,7 +977,7 @@ class BusinessChild(Page):
 
     subpage_types = []
     parent_page_types = ["tests.BusinessIndex", BusinessSubIndex]
-    page_description = lazystr("A lazy business child page description")
+    page_description = _("A lazy business child page description")
```

Comment:
I believe this one is done on purpose. `lazystr` is a shortcut for a callable that returns a str, which this page description is doing

Reply:
@zerolab that's actually my code line ;) See https://github.com/wagtail/wagtail/pull/8527#issue-1233696888
And yes, it had a purpose: get some lazy string analog to gettext_lazy without creating a new translation string, as it happened with other strings in the tests.
Now, with the exclusion of the tests from makemessages, this line can be "reverted", using a normal `gettext_lazy()`/`_()` call.